front-end/src/style.css

```diff
@@ -311,12 +340,14 @@ body {
 .player_join {
   text-align: center;
   height: 100%;
+  margin-top: 10px;
 }
 
 .banner {
-  background-color: rgba(117, 125, 232, 0.4);
+  background: rgb(169,101,203);
+  background: radial-gradient(circle, rgba(169,101,203,0.7259278711484594) 27%, rgba(79,19,198,1) 100%);
   font-weight: bold;
-  margin-top: 1vw;
+  margin-top: 0.05vw;
```

Comment:
not sure if we actually want this? I think the banner top should be flush with the top of the page. so I think we can just remove this line.